client/src/components/application/Transactions.vue

```diff
@@ -375,4 +403,9 @@
   .ma-0 {
     margin: 0 !important;
   }
+
+  /* Vuetify overrides: */
+  >>> .v-input__control {
```

Comment:
What does `>>>` mean?

Reply:
It's an older version of ::v-deep, more or less the same. ::v-deep wasn't working as intended in this version, for myself at least.